Mos/en.lproj/Main.strings

```diff
@@ -7,48 +7,48 @@
+"TUE-3v-CaJ.title" = "          Rendering this chart will affect scrolling performance. Close the window when not in use.";
 
 // PreferencesWindow
 // General
 "ITL-GD-CPd.title" = "General";
 "ILt-Gz-RiV.title" = "Scroll:";
 "weE-be-TAy.title" = "Smooth Scrolling";
-"G8a-97-8Ey.title" = "Let the mouse wheel scroll like a buttery.";
+"G8a-97-8Ey.title" = "Let the mouse wheel scroll smoothly like butter";
 "6FR-2r-rxl.title" = "Scroll Reverse";
-"VUb-ZM-hXo.title" = "Reverse the mouse wheel scroll direction.";
-"8sv-2I-VeW.title" = "Other Setting:";
+"VUb-ZM-hXo.title" = "Reverse the mouse wheel scroll direction";
+"8sv-2I-VeW.title" = "Miscellaneous:";
 "HZF-2o-Mrh.title" = "Launch on Login";
-"wY5-9W-iNv.title" = "Automaticly launch on MacOS login.";
+"wY5-9W-iNv.title" = "Automatically launch app on login";
 // Advanced
 "iW5-CD-5OC.title" = "Advanced";
-"NAp-3R-e8q.title" = "Shift Key:";
-"lzC-ub-pqJ.title" = "Transform the vertical scroll to horizontal";
+"NAp-3R-e8q.title" = "Toggle Key:";
+"lzC-ub-pqJ.title" = "Change vertical scrolling to horizontal scrolling";
 "akk-dX-tW4.title" = "Block Key:";
-"62p-76-okJ.title" = "Temporarily block the smoothing of the scroll";
+"62p-76-okJ.title" = "Temporarily block smooth scrolling";
 "bcp-Xx-rdX.title" = "Disable";
 "DXO-Xo-tJd.title" = "Disable";
 "YE2-UH-9Ku.title" = "Step:";
-"3Xk-SU-gqO.title" = "The minium scroll distance.";
+"3Xk-SU-gqO.title" = "Sets the minimum scroll distance";
 "wck-ay-cL9.title" = "Speed:";
-"PK6-dT-z93.title" = "The speed gain of scrollng.";
+"PK6-dT-z93.title" = "Sets scrolling acceleration";
 "suV-WY-rbw.title" = "Duration:";
-"vmk-zZ-ifN.title" = "The duration of scroll animation";
-"2ho-yL-X2k.title" = "Reset to default";
-"bAe-Zc-5SS.title" = "In some applications, scrolling is used to adjust certain precise parameters (such as adjusting the volume or zooming the document, etc.), and smooth scrolling may cause some interference to the operation. You can set a block key to perform such operations. Temporarily blocking the smoothing of the scroll.";
+"vmk-zZ-ifN.title" = "Sets the duration of the scroll animation";
+"2ho-yL-X2k.title" = "Reset to defaults";
+"bAe-Zc-5SS.title" = "In some applications, scrolling is used to adjust certain precise parameters (such as adjusting the volume, zooming the display, etc.) and smooth scrolling may cause interference. You can set a block key which will temporarily disable smooth scrolling when the key is pressed.";
 // Exception
-"I5j-Bv-Aiy.title" = "Need to using Accessibility features";
-"FU4-wI-K8f.title" = "To identify the exception application, we need to dynamically obtain the window information. Therefore, please allow us access to your accessibility feature.";
+"I5j-Bv-Aiy.title" = "Needs access to Accessibility feature";
+"FU4-wI-K8f.title" = "In order to custom the scroll behavior in certain applications, please allow access to Accessibility feature.";
```

Comment:
I misssed this but this should say "In order to customize..." not "custom". Do you want me to create a different pull request or would you like to fix it?

Reply:
Very welcome you to continue to create a PR !  😀😁